src/modules/displayfood.js

```diff
@@ -7,7 +7,7 @@ export const display = async () => {
 /* eslint-enable */
```

Comment:
We set up linters to assist us in writing standardized code. Please enable linters and try to resolve the issue. If a rule proved difficult to solve, you are only permitted to disable that rule. Even so, you'll need to determine whether the rule you're disabling is something that can be easily solved or not.